docs/convolution/index.rst

```diff
@@ -418,31 +418,6 @@ eye.
 ..
   EXAMPLE END
 
-.. _astropy_convolve_compat:
```

Comment:
I don't know why it didn't fail here but removal of this reference broke RTD build: `<unknown>:5546: WARNING: undefined label: astropy_convolve_compat`

Perhaps we can set up redirects but I am not sure how, since this is just a subsection in a page. The existing redirects are all `.html` or directory redirects. I think we need to find a workaround for this failure until @embray can work on a proper redirect.

Reply:
Actually, lemme dig to see what is trying to link to it in `main`. Hold on.

Reply:
Ah, it is in `CHANGES.rst` (L5552). Still, I wonder why RTD here did not trigger it before merge. 🤔

Reply:
I think the only sane way forward is to make an exception here to "change history" in removing the ref from `CHANGES.rst`. That would be easier than setting up redirect. @eerovaher , would you be interested to open a follow-up PR?

Reply:
So weird that it didn't fail in CI! It does give a bit of confirmation that our usual policy of not linking to anything in the changelog is actually good, even though I've often wondered about it...

Reply:
Maybe this is some `towncrier` dark magic of not rendering the change log in PRs or something? @saimn would know.

Reply:
I can see two options to deal with this.
1) Deleting the sentence that contains the offending link.
2) Making the link direct to version 2.0.16 documentation.

Which of the two is preferable?

It would also be a good idea to explicitly state in the developer documentation that the change log entries should not contain links.

Reply:
It seems indeed that the PR fragment is not taken into account when rendering the changelog. E.g. in https://astropy--11880.org.readthedocs.build/en/11880/changelog.html I cannot find the entry for #11880. Not sure why, maybe ping @Cadair ;)